### PR TITLE
Added documentation for recentPrinterShares API

### PR DESCRIPTION
**Pull request by venkatnagula**
The purpose of this PR is enable an API to allow signed user to fetch their recentprintershares in V1.0. currently it is rolled out to Beta and stable. 

  Moving it to V1.0 now. Below the links 

API.md : https://microsoftgraph.visualstudio.com/onboarding/_git/onboarding/pullrequest/9441  
PR to CSDL : https://dev.azure.com/msazure/One/_git/AD-AggregatorService-Workloads/pullrequest/8264884
Beta documentation PR: https://github.com/microsoftgraph/microsoft-graph-docs/pull/16144

### CODE REVIEW

**Review by braedenp-msft — APPROVED** [review]

File: api-reference/v1.0/resources/printershareviewpoint.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+# printerShareViewpoint resource type
+
+Namespace: microsoft.graph
+
+Represents user-specific data for a printer share as viewed by the signed-in user.
+
+## Properties
+|Property|Type|Description|
+|:---|:---|:---|
+|lastUsedDateTime|DateTimeOffset|Date and time when the printer was last used by the signed-in user. The Timestamp type represents date and time information using ISO 8601 format and is always in UTC time. For example, midnight UTC on Jan 1, 2014 is `2014-01-01T00:00:00Z`.|
```
Comment (Danielabom) [inline api-reference/v1.0/resources/printershareviewpoint.md:19]:
```suggestion
|lastUsedDateTime|DateTimeOffset|Date and time when the printer was last used by the signed-in user. The timestamp type represents date and time information using ISO 8601 format and is always in UTC time. For example, midnight UTC on Jan 1, 2014 is `2014-01-01T00:00:00Z`.|
```

File: api-reference/v1.0/resources/printershare.md [hunk 2]
```diff
@@ -45,6 +45,7 @@ Inherits from [printerBase](../resources/printerbase.md).
 |manufacturer|String|The manufacturer reported by the printer associated with this printer share. Inherited from [printerBase](../resources/printerbase.md). Read-only.|
 |model|String|The model name reported by the printer associated with this printer share. Inherited from [printerBase](../resources/printerbase.md). Read-only.|
 |status|[printerStatus](printerstatus.md)|The processing status, including any errors, of the printer associated with this printer share.Inherited from [printerBase](../resources/printerbase.md). Read-only.|
+|viewPoint|[printerShareViewpoint](../resources/printershareviewpoint.md)|Additional data for a printer share as viewed by the signed-in user.|
```
Comment (nilakhan) [inline api-reference/v1.0/resources/printershare.md:48]: do we want to add that this only be populated in GET /me/print/recentPrinterShares API

Comment (venkatnagula) [reply api-reference/v1.0/resources/printershare.md:48]: I am just reusing whatever is used in beta version. Let me know if you have any suggestion for better wording.

Comment (nilakhan) [reply api-reference/v1.0/resources/printershare.md:48]: haha. I am not the best with words either. @braedenp-msft thoughts?

**Review by nilakhan — APPROVED** [review]

**Pull request merged**